src/gluonnlp/data/word_embedding_evaluation.py

```diff
@@ -190,7 +202,7 @@ def _get_data(self):
                     self.root,
                     'wordsim353_sim_rel/wordsim_similarity_goldstandard.txt'))
 
-        return list({tuple(row) for row in CorpusDataset(paths)})
+        return sorted(list({tuple(row) for row in CorpusDataset(paths)}))
```

Comment:
Why sorted?

Reply:
it's generating a list from a set, which doesn't guarantee order. Doing a sort here could help make the result consistent and the doctest easier.

Reply:
I see.